.github/scripts/concretize-latest-spack.sh

```diff
@@ -0,0 +1,33 @@
+cd /
+latest=$(curl -s https://api.github.com/repos/spack/spack/releases/latest | grep '"tag_name":' | cut -d'"' -f4)
```

Comment:
Should this be an optional argument, so that we can also pass in a dedicated tag and only fall back to the latest if there is none?